packages/kuma-gui/src/app/connections/data/index.ts

```diff
@@ -4,7 +4,7 @@ const appProtocols = ['http', 'tcp', 'grpc'] as const
 // `_` followed by 1 to 5 digits followed by a `.`
 const trailingPortRe = /_\d{1,5}\./
 const trailingPortRe2 = /_\d{1,5}/
-const meshServiceRe = /_(mz|m|ext){1}svc_\d{1,5}$/
+const meshServiceRe = /_(mz|m|ext){1}svc_\d{1,5}(-[a-z0-9]+)?$/
```

Comment:
@lahabana should we make this forwards compatible now:

> should we use _ as separator for the suffix so it's compliant or do we do more complex regex matching?

https://github.com/kumahq/kuma-gui/issues/3237#issue-2701729992

Wasn't there a MADR somewhere outlining a "standard" format for these names?

Reply:
Also see https://github.com/kumahq/kuma/issues/12147 which we just made in relation to this